lib/ContextModule.js

```diff
@@ -65,6 +67,7 @@ class ContextModule extends Module {
 	}
 
 	identifier() {
+		if (this._identifier) return this._identifier;
 		let identifier = this.context;
```

Comment:
Move identifier creation into constructor like with the MultiModule.

Move it into a `_createIdentifier()` function and call this in the constructor.